### ISSUE TITLE
Everywhere "css stylesheet" is written it should say "class"

### ISSUE THREAD

**Issue by OscarGodson** [mikeplate/jquery-drag-drop-plugin#8]
Unless I'm wrong you specifically mean the class of the element, not the stylesheet name, right?

**Comment by redbeardcreator** [comment]
I agree. So, I've got a fix in the pipeline to fix it. I'll be putting in a pull request sometime in the next few days.
